examples/generated/test/react-dom/components/ComponentParameterInstance.js

```diff
@@ -10,6 +10,7 @@ export default class ComponentParameterInstance extends React.Component {
   render() {
 
 
+
```

Comment:
this is odd... i wonder why we're getting another newline now in so many places... is there an optional that defaults to newline for `None`, that's getting hit a lot more now?

Reply:
I previously had a special case in one of the list output functions in `javaScriptRender` that wasn't quite correct, but I left it for a while to reduce the extra newlines. I'll take a look at these in another PR